packages/graphql/mixin.server.js

```diff
@@ -19,6 +23,24 @@ const fetch = require('cross-fetch');
 
 let introspectionResult = undefined;
 
+const renderMarkupAndHelmetFragments = async (element, canPrefetchOnServer) => {
```

Comment:
Shouldn't the second argument rather be `shouldPrefetchOnServer` or just `prefetchOnServer`? The question whether we *can* prefetch has already been answered through the call to the mixin hook.

Reply:
Fixed with the latest force-push.